src/shared/cgroup-setup.c

```diff
@@ -445,6 +445,9 @@ int cg_enable(
                                                                            * not enabled. */
                                         ret |= bit;
                         } else {
+                                log_debug("%s controller %s for %s (%s).",
```

Comment:
log_trace? Otherwise this might become extremely noisy

Reply:
I just dropped the log message if you think it'll become noisy.